app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -21,14 +25,15 @@ use web_sys::Element;
 pub struct SideMenu {
     logger:             Logger,
     pub root_dom:       Element,
+    frp:                Frp,
     new_project_button: Element,
     projects_list:      Element,
     closures:           Rc<RefCell<Vec<ClickClosure>>>,
```

Comment:
does this need to be in RefCell? Can't we use &mut functions instead?

Reply:
I can't see any other way. `SideMenu` needs to be `CloneRef` so that `Model` would be `CloneRef`, so that `View` would be `CloneRef`. We need interior mutability somewhere: it might be `RefCell<Closures>` in `SideMenu`, as it is now, or `RefCell<SideMenu>` in `Model`.
Mutability is needed because the list of projects can be updated after the creation of `SideMenu`, which will cause creation of new `ClickClosure`s.